docs/swagger_v2/blocks.spec.yaml

```diff
@@ -1,5 +1,5 @@
 schemas:
-  Block:
+  KeyBlock:
```

Comment:
a change to the old endpoint name will work as "breaking" in terms of how the aepps consume them currently I guess

Reply:
maybe @davidyuk can give some input on that?

Reply:
changing `operationId` is a breaking change for apps that using `swagger-client`, yes